### DISCUSSION TITLE
Android install step 13 help

### DISCUSSION THREAD

**Discussion by pdfrench** [tesla-android/issue-tracker#22] [Q&A]
I get to step 13 in the Android install.

I type in adb root. This completes successfully.

I then enter: adb shell 'sqlite3 /data/data/com.google.android.gsf/databases/gservices.db "select * from main where name = \"android_id\";"'

but get the following error : inaccessible or not found

![image](https://user-images.githubusercontent.com/32956576/168590051-6bc1df10-d69d-460a-b1f0-b3ea96d83ff9.png)

**Comment by mikegapinski** [comment]
there has to be some kind of issue with the command, copy it from google(it's visible on the device registration website).

The Android ID is accessible after the error from play protect has been shown on the display, was this the case for you? The command won't return anything if you run in just after booting to Android
